mio/cli/stream.py

```diff
@@ -26,7 +26,8 @@ def _common_options(fn: Callable) -> Callable:
         "--device_config",
         required=True,
         help=(
-            "Either a config `id` or a path to device config YAML file for streamDaq "
+            "Either a config `id` or a path to device config YAML file for streamDaq. "
+            "This is currently the id located in the first line of the YAML config file "
```

Comment:
```suggestion
            "If you aren't using `id` you can ignore them."
```

Reply:
I'm still not exactly sure what additional information this gives because i'm not exactly sure what is ambiguous about the current message, i would figure this would be covered by the "either a config `id` or a path" where if i didn't want to use the id field i would use a path, but if this is clearer to someone who isn't me, then i'm fine with it

Reply:
Exactly, it does not add anything logic-wise! It just prevents someone from thinking, "What's this ID thing?" before we prepare a feasible way to figure it out (which you are preparing in #93 ).